api/libproject/devstart.rb

```diff
@@ -1669,10 +1661,8 @@ def load_config(project, dry_run = false)
 
   common = Common.new
   common.status "Loading #{config_json} into database..."
-  Dir.chdir("tools") do
-    run_inline_or_log(dry_run, %W{gradle --info loadConfig -Pconfig_key=main -Pconfig_file=../config/#{config_json}})
-    run_inline_or_log(dry_run, %W{gradle --info loadConfig -Pconfig_key=cdrBigQuerySchema -Pconfig_file=../config/cdm/cdm_5_2.json})
-  end
+  run_inline_or_log(dry_run, %W{gradle --info loadConfig -Pconfig_key=main -Pconfig_file=config/#{config_json}})
+  run_inline_or_log(dry_run, %W{gradle --info loadConfig -Pconfig_key=cdrBigQuerySchema -Pconfig_file=config/cdm/cdm_5_2.json})
```

Comment:
I can't comment directly on it, but I'm suspect of line 1743 (may need to drop the `../`).

You can test with `./project.rb deploy --project aou-db-test --no-promote --version gjordan --dry-run`, or you can drop the --dry-run for a more complete test, pushing an unpromoted version is not a problem.